File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/24_compito_java_2018_07_25/Progetto.java
import java.io.*;
import java.util.*;

public abstract class Progetto {
    protected int codice;
    protected String titolo, coord, org;
    protected double importo;

    public Progetto(int cod, String t, String c, String o, double i) {
        codice = cod;
        titolo = t;
        coord = c;
        org = o;
        importo = i;
    }

    public String getTitolo() {
        return titolo;
    }

    public int getCod() {
        return codice;
    }

    public String toString() {
        return titolo + "\t" + codice + "\t" + coord + "\t" + org + "\t";
    }
}

File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/24_compito_java 20180725/Progetto.py
class Progetto:
    def __init__(self, codice, tipo, titolo, nome, organizzazione, importo):
        self._codice = codice
        self._tipo = tipo
        self._titolo = titolo
        self._nome = nome
        self._organizzazione = organizzazione
        self._importo = importo

    def get_nome_progetto(self):
        return self._titolo

    def __str__(self):
        return self._tipo + "\t" + self._titolo + "\t" + self._nome + "\t" + self._organizzazione
